.github/workflows/release-production.yml

```diff
@@ -33,14 +41,25 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Build and push images
+      # Build the Docker image
+      - name: Build Docker image
+        id: docker_build
         uses: docker/build-push-action@v2
         with:
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
-          platforms: linux/amd64, linux/arm64
+          context: .
```

Comment:
Why these context changes

Reply:
I missed the original context settings when refactoring.  They are restored.